src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -106,171 +106,4 @@ public void equals() {
 
     }
 
-    /**
-     * A default model stub that have all of the methods failing.
-     */
-    private class ModelStub implements Model {
```

Comment:
Make sure that if the developer guide mentions using `ModelStub` in testing, it should be removed.